front-end/src/screens/PrivacyPolicy/index.tsx

```diff
@@ -0,0 +1,123 @@
+import styled from '@xstyled/styled-components';
+import React from 'react';
+import ReactMarkdown from 'react-markdown';
+import { Grid } from 'semantic-ui-react';
+
+interface Props {
+	className?: string
+}
+
+const PrivacyPolicy = ({ className }:Props): JSX.Element => {
+
+	return (
+		<Grid className={className}>
+			<Grid.Column className='policy_box' mobile={16} tablet={12} computer={10} largeScreen={10} widescreen={4}>
+				<h1>Privacy Policy</h1>
+				<ReactMarkdown className='markdown'>
+					## Placeholder MD
```

Comment:
It'd be easier to maintain using a dedicated external file. You can load it with `ReactMarkdown` using the `source` prop.
I'd place some dummy content in a `/public/privacy-pocily.md` for instance. Same for the t&cs.

Reply:
Thanks for the tip. I was aware of the source prop, but couldn't make it work as the .md file doesn't get compiled into a string. So I end up displaying the reference.
https://github.com/rexxars/react-markdown/issues/76

Tried with `raw-loader` as advised in the thread above, but no success.

Reply:
Solved with the `.ts` file approach, thanks :)